legal-api/src/legal_api/models/filing.py

```diff
@@ -159,16 +159,33 @@ class Source(Enum):
         },
         'dissolution': {
             'name': 'dissolution',
-            'title': 'Voluntary dissolution',
-            'codes': {
-                'CP': 'DIS_VOL',
-                'BC': 'DIS_VOL',
-                'BEN': 'DIS_VOL',
-                'ULC': 'DIS_VOL',
-                'CC': 'DIS_VOL',
-                'LLC': 'DIS_VOL',
-                'SP': 'DIS_VOL',
-                'GP': 'DIS_VOL'
+            'voluntaryDissolution': {
+                'name': 'voluntaryDissolution',
+                'title': 'Voluntary Dissolution',
+                'codes': {
+                    'CP': 'DIS_VOL',
+                    'BC': 'DIS_VOL',
+                    'BEN': 'DIS_VOL',
+                    'ULC': 'DIS_VOL',
+                    'CC': 'DIS_VOL',
+                    'LLC': 'DIS_VOL',
+                    'SP': 'DIS_VOL',
+                    'GP': 'DIS_VOL'
+                }
+            },
+            'administrativeDissolution': {
+                'name': 'administrativeDissolution',
+                'title': 'Administrative Dissolution',
+                'codes': {
+                    'CP': 'DIS_ADM',
+                    'BC': 'DIS_ADM',
+                    'BEN': 'DIS_ADM',
+                    'ULC': 'DIS_ADM',
+                    'CC': 'DIS_ADM',
+                    'LLC': 'DIS_ADM',
+                    'SP': 'DIS_ADM',
+                    'GP': 'DIS_ADM'
+                }
```

Comment:
Why do we have a duplicate struture here (same as file above)?

Reply:
yes, the duplication is not ideal.  I think the filing meta was intended for meta data like display name.  but it does include things like fee codes which are found in the filing model as well.  I'm not sure the story behind that.

Reply:
OK. 🙈

Reply:
i know...